api/src/main/java/com/github/retrooper/packetevents/protocol/item/jukebox/IJukeboxSong.java

```diff
@@ -43,12 +43,10 @@ public interface IJukeboxSong extends MappedEntity, CopyableEntity<IJukeboxSong>
 
     static IJukeboxSong decode(NBT nbt, ClientVersion version, @Nullable TypesBuilderData data) {
         NBTCompound compound = (NBTCompound) nbt;
-
-        Sound sound = Sound.decode(compound.getCompoundTagOrThrow("sound_event"), version);
+        Sound sound = Sound.decode(compound.getStringTagOrThrow("sound_event"), version);
```

Comment:
should be changed to just `getTagOrThrow`, this can both be a compound or a string